app/controllers/course/assessment/submission/submissions_controller.rb

```diff
@@ -244,7 +257,16 @@ def create_params
   end
 
   def create_success_response(submission)
-    redirect_url = edit_course_assessment_submission_path(current_course, @assessment, submission)
+    is_course_koditsu_enabled = current_course.component_enabled?(Course::KoditsuPlatformComponent)
+    is_assessment_koditsu_enabled = @assessment.koditsu_assessment_id && @assessment.is_koditsu_enabled
+
+    if is_course_koditsu_enabled && is_assessment_koditsu_enabled
+      submission.create_new_answers
+      redirect_url = "https://code.codaveri.com?assessment=#{@assessment.koditsu_assessment_id}"
```

Comment:
Hard-coded string instead of using ENV variable.  
Will break or cause unexpected behaviour once we move to production.

Reply:
fixed